specification/app/Microsoft.App.DynamicSessions/tspconfig.yaml

```diff
@@ -12,26 +12,26 @@ options:
     emitter-output-dir: "{project-root}/.."
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/DynamicSessions.json"
   "@azure-tools/typespec-python":
-    package-dir: "microsoft-app-dynamicsessions"
+    package-dir: "azure-microsoft-app-dynamicsessions"
     package-name: "{package-dir}"
     package-mode: dataplane
     flavor: azure
   "@azure-tools/typespec-csharp":
-    package-dir: "Microsoft.App.DynamicSessions"
+    package-dir: "Azure.Microsoft.App.DynamicSessions"
     namespace: "{package-dir}"
     clear-output-folder: true
     model-namespace: false
     flavor: azure
   "@azure-tools/typespec-ts":
-    package-dir: "microsoft-app-dynamicsessions-rest"
+    package-dir: "azure-microsoft-app-dynamicsessions-rest"
```

Comment:
why change this?

Reply:
Was trying to address the following error but looks like it made it worse.
``` 
 - The value of options.@azure-tools/typespec-java.package-dir "microsoft-app-dynamicsessions" does not match "/^azure(-\w+)+$/". Please update the value of "options.@azure-tools/typespec-java.package-dir" to match "/^azure(-\w+)+$/".
  - The value of options.@azure-tools/typespec-csharp.package-dir "Microsoft.App.DynamicSessions" does not match "/^Azure\./". Please update the value of "options.@azure-tools/typespec-csharp.package-dir" to match "/^Azure\./".
  ```

Reply:
Okay I didn't know what this change would do, so I reverted it.

Reply:
@najian do I need one more approval to get this PR merged?